f/frizzle/kobotoolbox/kobotoolbox_responses.script.yaml

```diff
@@ -1,4 +1,4 @@
-summary: 'Kobo: Fetch Survey Responses'
+summary: 'KoboToolbox: Fetch Survey Responses'
 description: Uses KoboToolbox REST API to download all form submissions
 lock: '!inline f/frizzle/kobo/kobo_responses.script.lock'
```

Comment:
We missed this line.  I fixed in https://github.com/ConservationMetrics/gc-scripts-hub/commit/1aa499b223a926b87d3f2fbbdcd2bc0592308898

Reply:
thanks.